CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -601,38 +601,38 @@ function Patient:tickDay()
   -- TODO clean up this block, nonmagical numbers
   local health = self:getAttribute("health")
   if health >= 0.18 and health < 0.22 then
-    self:setMood("sad2", "activate")
+    self:setMood("dying1", "activate")
     self:changeAttribute("happiness", -0.0002) -- waiting too long will make you sad
     -- There is a 1/3 chance that the patient will get fed up and leave.
     -- This is potentially run 10 ((0.22-0.18)/0.004) times, hence the 1/30 chance.
     -- If patient is already in the cure room, let the treatment happen.
     if not self:_checkIfCureRoom(self:getRoom()) and math.random(1,30) == 1 then
       self:setDynamicInfoText(_S.dynamic_info.patient.actions.fed_up)
-      self:setMood("sad2", "deactivate")
+      self:setMood("dying1", "deactivate")
       self:goHome("kicked")
     end
   elseif health >= 0.14 and health < 0.18 then
-    self:setMood("sad2", "deactivate")
-    self:setMood("sad3", "activate")
+    self:setMood("dying1", "deactivate")
+    self:setMood("dying2", "activate")
   -- now wishes they had gone to that other hospital
   elseif health >= 0.10 and health < 0.14 then
-    self:setMood("sad3", "deactivate")
-    self:setMood("sad4", "activate")
+    self:setMood("dying2", "deactivate")
+    self:setMood("dying3", "activate")
   -- starts to take a turn for the worse and is slipping away
   elseif health >= 0.06 and health < 0.10 then
-    self:setMood("sad4", "deactivate")
-    self:setMood("sad5", "activate")
+    self:setMood("dying3", "deactivate")
+    self:setMood("dying4", "activate")
   -- fading fast
   elseif health >= 0.01 and health < 0.06 then
-    self:setMood("sad5", "deactivate")
-    self:setMood("sad6", "activate")
+    self:setMood("dying4", "deactivate")
+    self:setMood("dying5", "activate")
   -- it's not looking good
   elseif health > 0.00 and health < 0.01 then
     self.attributes["health"] = 0.0
   -- is there time to say a prayer
   elseif health == 0.0 then
     if not self:getRoom() and not self:getCurrentAction().is_leaving then
-      self:setMood("sad6", "deactivate")
+      self:setMood("dying5", "deactivate")
```

Comment:
Not your fault, but the witty remarks aren't useful, and I feel this should be a table of conditions and corresponding moods instead of this if-with-many-alternatives statement.

I can live with not changing it. (Technically it's also out of scope.)